environment-3.8.yml

```diff
@@ -0,0 +1,37 @@
+name: alertadengue
+channels:
+  - conda-forge
+dependencies:
+- celery
+- dbfread
+- dj-database-url
+- dj-static
+- django 3.*
```

Comment:
We have a vulnerability in the current version of django. See #418; It solves with 3.1.8 version.
You can trigger a rebase manually by commenting @dependabot rebase.